scipy/spatial/transform/_rotation.pyx

```diff
@@ -2766,12 +2766,8 @@ class Slerp:
 
     """
     def __init__(self, times, rotations):
-        if rotations.single:
-            raise ValueError("`rotations` must be a sequence of rotations.")
-
-        if len(rotations) == 1:
-                raise ValueError("`rotations` must contain at least 2 "
-                                 "rotations.")
+        if rotations.single or len(rotations) == 1:
+            raise ValueError("`rotations` must be a sequence of at least 2 rotations.")
```

Comment:
I did not check but we usually have tests checking the actually message from raises. If this does not trigger anything, that would be a good sign showing a test could and should be added.

Reply:
Okay, gotcha. I just combined two checks/errors that were doing the same thing for the sake of avoiding repetition.

Reply:
Actually if we want to really improve the input validation here, adding a check on the type of `rotations` would be valuable I think. (This would also need a test)